documentation/asciidoc/topics/attributes/product-attributes.adoc

```diff
@@ -1,77 +1,111 @@
 //
-// This file contains attributes for building Red Hat Data Grid documentation.
+// Attributes for building Red Hat Data Grid documentation.
 //
 
 //
-// Attributes Shared with Community Docs
+// Operator resources
 //
-:brandname: Data Grid
-:wildflybrandname: EAP
-:tomcatbrandname: JWS
-:infinispanversion: 7.3.0
-
-// Community Operator
-
-:ispn_operator: Data Grid Operator
-:k8s: OpenShift
-:okd: OpenShift Container Platform
-
 :github_root: https://github.com/infinispan/infinispan-operator/
 :github_raw: https://raw.githubusercontent.com/infinispan/infinispan-operator/
 :github_tag: master
 
 :example_crd_name: example-rhdg
 
-// Community Doc Roots
-:javadocroot: https://access.redhat.com/webassets/avalon/d/red-hat-data-grid/7.3/api
-:configdocroot: https://access.redhat.com/webassets/avalon/d/red-hat-data-grid/7.3/Configuration/
-:wildflydocroot: https://docs.jboss.org/author/display/WFLY11
-:jdkdocroot: https://docs.oracle.com/javase/8/docs/api
-:javaeedocroot: https://docs.oracle.com/javaee/7/api
-
 //
-// Define conditional statement for product docs
+// Conditional statements
 //
 :productized:
+:server:
+:library:
 
 //
-// RH Product Names and Versions
+// Directory attributes
 //
+:images_dir: images
+:stories: stories
+:topics: topics
+
+//
+// Product names and versions
+// These attributes should always be conditional for product only
+//
+:RedHat: Red Hat
+:portal: Red Hat customer portal
+
 :FullProductName: Red Hat Data Grid
 :ProductName: Data Grid
 :ProductShortName: RHDG
-:ProductEAP: Jboss Enterprise Application Platform (EAP)
-:ProductJWS: JBoss Web Server (JWS)
-:ProductRelease: 7.3
-:ProductVersion: 7.3
-:FullProductRelease: 7.3.1
-:FullProductVersion: 7.3.1
-:DocInfoProductNumber: 7.3
-:ProductPreviousVersion: 7.2
+:ProductVersion: 8.0
+
+:ProductRelease: 8.0
+:FullProductRelease: 8.0.0
+:FullProductVersion: 8.0.0
+:DocInfoProductNumber: 8.0
+:ProductPreviousVersion: 7.3
 :ProductNextVersion: 8.0
-:ProductPreviousRelease: 6
-:ProductPreviousPreviousRelease: 5
-:plusplus: ++
-:underscores: ___
+:ProductPreviousRelease: 7
+:ProductPreviousPreviousRelease: 6
 
 //Doc Home
 :doc_home: https://access.redhat.com/documentation/en-us/red_hat_data_grid/
 
-//HR Docs URL for Product
-:hr_url: https://access.redhat.com/webassets/avalon/d/red-hat-data-grid/7.3/java/org/infinispan/client/hotrod/
+//
+// Special characters
+//
+:plusplus: ++
+:underscores: ___
+
+//
+// Community attributes
+// In the main Infinispan repo, these attributes are defined in
+// documentation/pom.xml
+//
+:brandname: Data Grid
+:fullbrandname: Red Hat Data Grid
+:wildflybrandname: EAP
```

Comment:
Can these wildfly references be removed? They're not relevant nor to the server nor the operator any more.

Reply:
@galderz I need to investigate before removing. This wildfly attribute is in the community docs as well. 

https://github.com/infinispan/infinispan/blob/68170ad67914ac29ee3194c3e9e7c2e006c6460d/documentation/pom.xml#L170 

And here is an example of the attribute in the docs: https://github.com/infinispan/infinispan/blob/master/documentation/src/main/asciidoc/topics/integrations.adoc#server-mode 

I'd much rather remove this attribute as part of a larger effort to scrub wildfly/eap references across the entire doc set. I've created ISPN-10564 to handle that.

Reply:
@galderz ignore the previous comment. I was thinking about the main Infinispan repo. I've removed those wildfly attributes. cheers.